mezzanine/core/admin.py

```diff
@@ -8,7 +8,8 @@
 from django.contrib.auth.models import User as AuthUser
 from django.contrib.redirects.admin import RedirectAdmin
 from django.contrib.messages import error
-from django.core.urlresolvers import NoReverseMatch
+from mezzanine.compat import NoReverseMatch
+# from django.core.urlresolvers import NoReverseMatch
```

Comment:
As @stephenmcd said already you should not leave commented code. In your case you should just replace the line with it's new version. If you want to see the old version of that line there is GIT. It's exactly for that reason.